website/docs/components/table/partials/guidelines/guidelines.md

```diff
@@ -20,6 +20,24 @@
 - Columns that do contain a sortable data type are interactive and therefore have corresponding hover, active, and focus states.
 - A Table may only be sorted by a single value at a time.
 
+### Tooltips
+
+Labels within the Table column should be clear, concise, and straightforward. In the case that more context or details are necessary a [Tooltip](/components/tooltip) can be used in conjunction with the label, but should be used sparingly and as a last resort.
+
+![Tooltips in a Header Column](/assets/components/table/table-tooltip-example.png)
+
+!!! Dont
+
+We recommend against using a tooltip in all or most columns in a table as this can add unnecessary visual clutter and increase the cognitive load on the user.
```

Comment:
Is there an alternative we should suggest? Something to guide them to a different solution/help them avoid this scenario?

Reply:
This is the part that I'm struggling with a bit. A couple of things I've considered (a bit multi-faceted):
- Providing more context around/associated with the table. This could be descriptive text (maybe even in the PageHeader), or use the flyout as a vehicle for communicating more complex labels/concepts
- There's probably an enhancement that can be made in the flow prior to the user arriving at a table that might make this content more clear.
- Adding a sort of key or legend element associated with the table. This could also potentially exist within a flyout.

Reply:
Hmm, yea, it seems there are some different alternative options depending on the use case. We can always address it later if we decide on more standardized guidance in the future.